source/slang/hlsl.meta.slang

```diff
@@ -5023,8 +5259,13 @@ When generating code for other targets, this parameter is ignored and has no eff
 **/
 struct $(item.name)
 {
+    /// Decrements the object's hidden counter.
```

Comment:
Add that not implemented for HLSL targets.

Reply:
ack